src/__tests__/loginReducers/login.reducer.test.js

```diff
@@ -0,0 +1,33 @@
+import expect from 'expect';
+import LOGIN from '../../constants/login.type';
+import loginReducer from '../../reducers/login.reducer';
+
+
+const initialState = {
+  login: {
+    onPending: true,
+    onFulfilled: true,
+    onRejected: true,
+  },
+};
+
+const action = { data: {} };
+
+describe('User Reducer test', () => {
+  it('should return initial state when there is no action', () => {
+    expect(loginReducer(initialState, action)).toEqual(initialState);
+  });
+
+  it('should handle USER_LOAD_REQUEST', () => {
+    action.type = `${LOGIN}_PENDING`;
+    expect(loginReducer(initialState.login, action).onPending).toEqual(true);
+  });
+  it('should handle USER_LOAD_REQUEST', () => {
```

Comment:
Have a blank line to separate each of your `it` blocks.
Apply this in all your tests